src/main/java/uk/gov/justice/digital/hmpps/cmd/api/domain/ShiftNotificationType.kt

```diff
@@ -7,8 +7,13 @@ enum class ShiftNotificationType(val value: String, val description: String) {
     SHIFT("shift", "shift"),
     OVERTIME("overtime", "overtime shift"),
     SHIFT_TASK("shift_task", "activity"),
+    TASK("task", "activity"), // The legacy code returns 'task' not 'shift_task'
```

Comment:
Could this duplication be replaces with a migration to replace the old values?

Reply:
No because its in prod and still sending them, there are 0 in the database, we can remove it as soon as we move over to the kotlin CSR app. If you look where this is used, its used to identify them and map to them to Shift_task before they're saved.